atlasdb-cli/src/main/java/com/palantir/atlasdb/cli/command/KvsMigrationCommand.java

```diff
@@ -171,22 +173,25 @@ public AtlasDbServices connectToServices() throws IOException {
         return DaggerAtlasDbServices.builder().servicesConfigModule(scm).build();
     }
 
-    private KeyValueServiceMigrator getMigrator(AtlasDbServices fromServices, AtlasDbServices toServices)
+    @VisibleForTesting
```

Comment:
This method could probably be moved to a factory class, and tested there, so we wouldn't need `@VisibleForTesting`.